README.md

```diff
@@ -237,6 +237,55 @@ spec:
 
 Create a PVC using the storage class created for the ZFS driver.
 
+1. If storageclass has fstype as ext4/3/2, xfs, btrfs then we have to create the PVC with storage size which is multiple of volblocksize mentioned in the storageclass. The same is also true if fstype is not mentioned in the storageclass in which case dafault will be ext4.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: openebs-zfspv
+parameters:
+  volblocksize: "4k"
+  fstype: "ext4"
+  poolname: "zfspv-pool"
+provisioner: zfs.csi.openebs.io
+```
+
+Here, PVC should have the storage multiple of volblocksize which is 4k (4 * 1024). If is not the multiple of volblocksize then ZFS will not create the volume and pod will not be able to come into RUNNING state as mount will keep on failing. For example, if we have created the below PVC with the above storageclass
```

Comment:
This one needs an update based on the implementation change. https://github.com/openebs/zfs-localpv/pull/191

Reply:
updated @kmova